src/entrypoint/exporter.ts

```diff
@@ -1,12 +1,12 @@
 import express from "express"
 import client, { register } from "prom-client"
+import { WalletFromRole } from "src/walletFactory"
```

Comment:
I guess is because new ts build config that the IDE is using `src/` please use relative import

Reply:
what has been decided for the import (relative/absolute). can we write this down in the repo somewhere?